src/test/scala/mesosphere/marathon/api/v2/ModelValidationTest.scala

```diff
@@ -103,7 +103,7 @@ class ModelValidationTest
       case f: Failure =>
         val errors = (Json.toJson(f) \ "details").as[Seq[JsObject]]
         errors should have size 1
-        (errors.head \ "path").as[String] should be("/groups(0)/apps(1)")
+        (errors.head \ "path").as[String] should be("/groups(group1)/apps(invalid)")
```

Comment:
Thanks for the heads up @kolloch . Yeah,  the UI currently expects an index in the form of `(N)`, for example https://github.com/mesosphere/marathon-ui/blob/master/src/js/stores/AppFormStore.js#L135 . What kind of string values are you proposing here, @aquamatthias ? 
What do `groupN`, `invalid` etc. relate to?